src/mono/CMakeLists.txt

```diff
@@ -410,8 +411,7 @@ elseif(TARGET_ARCH STREQUAL "arm64")
 elseif(TARGET_ARCH MATCHES "arm")
   set(TARGET_ARM 1)
   set(MONO_ARCHITECTURE "\"arm\"")
-  # FIXME:
-  add_definitions("-DARM_FPU_VFP=1")
+  add_definitions("-DARM_FPU_VFP_HARD=1")
   set(TARGET_SIZEOF_VOID_P 4)
```

Comment:
Is this true for every arm system ?

Reply:
It looks like you're right that this change isn't accurate - Android's 32-bit ARM port uses the soft float ABI & indeed needs ARM_FPU_VFP. I'm trying to determine the correct behaviour for iOS. linux-arm RID should be using ARM_FPU_VFP_HARD.

Reply:
arm-linux-gnueabihf target, which we use, explicitly requires ARM_FPU_VFP_HARD. ARM_FPU_VFP or ARM_FPU_NONE could both be valid choices for arm-linux-gnueabi